src/components/create.rs

```diff
@@ -17,11 +21,19 @@ use crate::{
 };
 
 use super::{
-    BitcoinInput, ContinueButton, CopyButton, DerivedAddressOutput, FeeRateInput, Footer,
+    BitcoinInput, ContinueButton, CopyButton, DerivedAddressOutput, FeeRateSelector, Footer,
     NetworkInput, NpubInput, NpubInputDerivedAddress, PrimaryButton, TimelockInput,
     TransactionOutput, TxidInput,
 };
 
+/// Fetch fee rates from the configured Esplora endpoint.
+async fn fetch_fee_estimates() -> Result<HashMap<u16, f64>, Error> {
+    let esplora_client = create_client(&ESPLORA_ENDPOINT.read()).unwrap();
+    let fee_rates = get_fee_estimates(&esplora_client).await.unwrap();
+
+    Ok(fee_rates)
+}
```

Comment:
Use the one in utils: https://github.com/storopoli/scrow/blob/b21f93dab8523e24f8148d43cd9c48bc016e84e1/src/esplora.rs#L19-L25